lib/screen/boarding_screen.dart

```diff
@@ -0,0 +1,136 @@
+import 'package:flutter/material.dart';
+import 'package:sochem/screen/slide.dart';
+
+class BoardingPage extends StatefulWidget {
+  @override
+  _BoardingScreenState createState() => _BoardingScreenState();
+}
+
+class _BoardingScreenState extends State<BoardingPage> {
+  int _currentPage = 0;
+  List<Slide> _slides = [];
+  PageController _pageController = PageController();
+
+  @override
+  void initState() {
+    _currentPage = 0;
+    //put slide content here
+    _slides = [
+      Slide("assets/img1.png", "Replace this text with heading of first slide",
+          "Replace this text with content of first slide"),
+      Slide("assets/img2.png", "Replace this text with heading of second slide",
+          "Replace this text with content of second slide"),
```

Comment:
Keep the images into a directory called **slider** under the assests directory, for eg the img1 should be under `assets/slider/img1.jpg`. Other than that mention the path of slider image in a variable inside the constants.dart and then use that variable here in order to avoid any issue while changing the images in future.